modules/howtos/pages/encrypting-using-sdk.adoc

```diff
@@ -208,3 +208,19 @@ include::example$EncryptingUsingSDK.java[tag=legacy_decrypters,indent=0]
 NOTE: The legacy decrypters require a mapping function.
 For AES, this function accepts an encryption key name and returns the corresponding signing key name.
 For RSA, this function accepts a public key name and returns the corresponding private key name.
+
+=== Reading Unencrypted Fields
+
+From Java SDK 3.2.1, the `@Encrypted` annotation can now be used to migrate an existing field from unencrypted to encrypted. 
+If you annotate a field with:
+
+[source,java]
+----
+@Encrypted(migration = Encrypted.Migration.FROM_UNENCRYPTED)
+----
+
+then either encrypted or unencrypted values will be accepted during deserialization.
```

Comment:
is "accepted" the right word, if it's *de*serialization? I'm confused, so may be misreading this.

Reply:
Good question.
It's "accepted" because, before 3.2.1, unencrypted fields would cause the deserialiser to error out.
Let me think a bit about the wording.